src/org/typo3/chefci/v2/Pipeline.groovy

```diff
@@ -85,10 +86,14 @@ class Pipeline implements Serializable {
     }
 
     void execute() {
+        (new Slack(script)).buildStart()
```

Comment:
hm - the `new` makes no sense here... why not having `slack` as a property and doing

    slack = new Slack(script)

in the constructor - so here it's only `slack.buildStart()` and `slack.buildFinish()` afterwards? Think that's a cleaner solution :)